src/ControlsView.elm

```diff
@@ -42,49 +43,44 @@ view model =
 viewUiButtons : Model -> Html Msg
 viewUiButtons model =
     let
-        toggleNewSessionClass =
-            if model.showNewSessionUi && model.idOfSessionBeingEdited == Nothing then
-                "btn btn-default active"
-            else
-                "btn btn-default"
-
         toggleNewColumnClass =
             if model.showNewColumnUi then
-                "btn btn-default active"
+                "prog-bar__button-dropdown prog-bar__button-dropdown--active"
             else
-                "btn btn-default"
+                "prog-bar__button-dropdown"
 
         toggleNewTrackClass =
             if model.showNewTrackUi then
-                "btn btn-default active"
+                "prog-bar__button-dropdown prog-bar__button-dropdown--active"
             else
-                "btn btn-default"
+                "prog-bar__button-dropdown"
 
         toggleManageDatesClass =
             if model.showManageDatesUi then
-                "btn btn-default active"
+                "prog-bar__button-dropdown prog-bar__button-dropdown--active"
             else
-                "btn btn-default"
+                "prog-bar__button-dropdown"
 
         publishButtonText =
             if model.published then
                 "Unpublish"
             else
                 "Publish"
     in
-        div [ class "btn-toolbar", attribute "role" "toolbar", style [ ( "margin", "3rem" ) ] ]
-            [ div [ class "btn-group btn-group-lg", attribute "role" "group" ]
-                [ button [ class toggleNewSessionClass, type_ "button", onClick ToggleNewSessionUi ]
-                    [ text "New Session" ]
+        div [ class "prog-bar" ]
+            [ div []
+                [ button [ class "button button--new prog-bar__button", type_ "button", onClick ToggleNewSessionUi ]
+                    [ text "+ New Session" ]
                 , button [ class toggleNewTrackClass, type_ "button", onClick ToggleNewTrackUi ]
-                    [ text "New Track" ]
+                    [ text "Manage Tracks" ]
                 , button [ class toggleNewColumnClass, type_ "button", onClick ToggleNewColumnUi ]
-                    [ text "New Column" ]
+                    [ text "Manage Columns" ]
                 , button [ class toggleManageDatesClass, type_ "button", onClick ToggleManageDatesUi ]
                     [ text "Manage Dates" ]
-                , a [ href ("/events/" ++ model.eventId ++ "/programme-builder/preview"), target "_blank", type_ "button" ]
+                , a [ class "button button--secondary prog-bar__button", href ("/events/" ++ model.eventId ++ "/programme-builder/preview"), target "_blank", type_ "button" ]
```

Comment:
Thanks @harrygfox. I've followed your instructions but now it looks like:

![screen shot 2017-03-13 at 16 56 27](https://cloud.githubusercontent.com/assets/11833296/23865582/0d1603d0-080e-11e7-9210-82eecf445a8b.png)